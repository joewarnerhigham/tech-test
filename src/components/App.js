import React from 'react';
import '../styles/App.css';
import Search from './Search';

function App() {
  return (
    <div className="App" data-testid='main-div'>
      <Search />
    </div>
  );
}

export default App;
