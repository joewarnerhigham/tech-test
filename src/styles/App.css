body{
    display: flex;
    justify-content: center;
    height: max-content;
    padding: 30px;
    background-image: url('https://time.com/wp-content/uploads/2017/10/229-westerlund-21.jpg?w=2000');
}