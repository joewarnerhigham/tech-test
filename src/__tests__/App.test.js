import React from 'react';
import {render, cleanup} from '@testing-library/react';
import App from '../components/App';


describe('App', () => {

  afterEach(cleanup);
  
 it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
   })
});