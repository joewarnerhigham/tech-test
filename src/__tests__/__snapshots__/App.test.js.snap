// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
    data-testid="main-div"
  >
    <div
      class="container"
    >
      <img
        alt="nasa-logo"
        class="logo"
        data-testid="logo"
        src="nasa.jpg"
      />
      <input
        class="search-bar"
        data-testid="search-bar"
        placeholder="Search"
        type="text"
        value=""
      />
    </div>
  </div>
</DocumentFragment>
`;
